src/Axes/cartesianAxes.ts

```diff
@@ -418,12 +447,23 @@ export class CartesianAxes extends Axes {
           .text(this.options.y[0].unit)
       }
       if (this.options.y[1]?.label) {
-        labelGroup.append('text')
-          .attr('class', 'y1 label')
-          .attr('x', this.width)
-          .attr('y', -9)
-          .attr('text-anchor', 'end')
-          .text(this.options.y[1].label)
+        if (this.options.y[1].labelOrientation === LabelOrientation.Vertical) {
+          const offset = this.options.y[1].labelOffset ?? 0
+          labelGroup.append('text')
+            .attr('class', 'y1 label')
```

Comment:
This part, and later on the `.text(this.options.y[1].label)` is used in both cases. Can that be taken outside of the if-statement?

Reply:
Moved code up 1 level